go.mod

```diff
@@ -81,6 +82,25 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require (
```

Comment:
Please keep only 2 "require" blocks: one for direct and one for indirect. It helps minimizing merge conflicts and keeping things tidy.

Reply:
@ItalyPaleAle  done.. sorry about that